browser/src/dom/NotebookbarAccessibility.js

```diff
@@ -414,7 +414,7 @@ var NotebookbarAccessibility = function() {
 
 	this.initialize = function() {
 		setTimeout(function() {
-			if (window.mode.isDesktop()) {
+			if (window.mode.isDesktop() && !this.initialized) {
```

Comment:
what about switching the mode few times?
tabbed interface -> compact interface -> tabbed interface

it will not be reinitialized again probably?
this is called in Control.UIManager.js

Reply:
possibly we need to "reset" it on switching to classic mode / closing tabbed interface

Reply:
> what about switching the mode few times? tabbed interface -> compact interface -> tabbed interface
> 
> it will not be reinitialized again probably? this is called in Control.UIManager.js

I have no idea, how coded/design here. I just saw an inconsistency here. It looks like a life cycle issue. I supposed
it is a singleton.

Reply:
yes, view -> compact mode / tabbed mode

Reply:
Ok, you designed the control. I just did a quick fix, no depth analysis here, I have spotted a serious issue here. It is up to you guys to decide what is best.